pkg/universe.dagger.io/x/vgjm456@qq.com/helm/helm.cue

```diff
@@ -0,0 +1,93 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	// "dagger.io/dagger/core"
+
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/bash"
+	// "universe.dagger.io/docker"
+)
+
+#Install: {
+	// Name of your release
+	name:       string | *""
+	kubeconfig: dagger.#Secret
+
+	source: *"repository" | "URL" | "local"
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+	}
+
+	if source == "URL" {
+		URL: string
+	}
+
+	// -----
+
+	if source == "repository" {
+		chart:      string
+		repository: string
+		env: {
+			REPOSITORY: repository
+			CHART:      chart
+		}
+		script: contents: #"""
+				helm repo add somerepo $REPOSITORY
+				helm repo update
+				helm install \#(name) somerepo/$CHART \#(_generateName)
```

Comment:
Same there for `somerepo` and `_generateName` should be sent as environment value